### DISCUSSION TITLE
MLFlow Operator

### DISCUSSION THREAD

**Discussion by felipemoz** [apache/airflow#24385] [General]
**Description**
Helping MLOps team to tigger ML jobs after Airflow steps (new operator)

**Use case / motivation**
You can trigger mlflow jobs like sagemaker operator

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by turbaszek** [comment]
Hi @felipemoz would you like to work on it? This sounds like an interesting feature. I am happy to help if you decide to implement those operators :)

**Comment by subkanthi** [comment]
Hi @turbaszek , @felipemoz , Is this something I can tackle, please let me know.

**Comment by turbaszek** [comment]
@subkanthi feel free to open a PR :)

**Comment by subkanthi** [comment]
@felipemoz for this task, were u thinking of using the Mlflow REST API to create runs? I see it very similar to the databricks REST API to start jobs. Im still trying to figure out how to pass the python file location or the model in REST API.

**Comment by rd16395p** [comment]
Was there any movement on this? I can help code this feature.

**Comment by subkanthi** [comment]
Hi @rd16395p ,I wasnt clear on what needs to be added, since there is already a MLflow operator, please proceed if u have clarity.

**Comment by rd16395p** [comment]
@subkanthi Sure! I can try. I was thinking a few functions, such as the one commented above with sagemaker integration. I can see that being useful from my own experience. What do you think?

**Comment by subkanthi** [comment]
Hey @rd16395p , I have some time now, I can work on this, please let me know.
@turbaszek can you please assign this back to me if @rd16395p didnt get started.

**Comment by rd16395p** [comment]
@subkanthi Why don't we work together on it?

**Comment by vijaya-lakshmi-venkatraman** [comment]
Hello,
Is this still available?

**Comment by rd16395p** [comment]
Hello! 

Sorry about the very long delay - yes! Lets work together on this during the weekend.

**Comment by Superskyyy** [comment]
Hello, is there any new progress on this?

**Comment by potiuk** [comment]
@Superskyyy - I think the easiest way is to get it done is to work on it. For now I will move it discussion (where it belongs) but if you would like to contribute (as one of > 2000 contributors) and male a PR to get it implemented, that would be cooll

**Comment by adamwrobel-ext-gd** [comment]
Hey guys, any progress on this topic? Are you looking for contributors?

**Reply by Taragolis** [reply]
Still the same, no one create a PR `¯\_(ツ)_/¯`

**Comment by timonviola** [comment]
There is a provider package by Astronomer https://github.com/astronomer/airflow-provider-mlflow/
